backend/api/src/update-market.ts

```diff
@@ -49,10 +81,39 @@ export const updatemarket = authEndpoint(async (req, auth) => {
     )
     log('updated visibility')
   }
+  if (question) {
+    await firestore.doc(`contracts/${contractId}`).update({
+      question,
+    })
+    log('updated question')
+  }
+  let resolvedDescription: JSONContent | undefined = undefined
+  if (
+    description ||
+    descriptionHtml ||
+    descriptionMarkdown ||
+    descriptionJson
+  ) {
+    resolvedDescription = getDescriptionJson(
+      description,
+      descriptionHtml,
+      descriptionMarkdown,
+      descriptionJson
+    )
+    await firestore.doc(`contracts/${contractId}`).update({
+      description: resolvedDescription,
+    })
+    log('updated description')
+  }
   await recordContractEdit(
     contract,
     auth.uid,
-    buildArray([visibility && 'visibility', closeTime && 'closeTime'])
+    buildArray([
+      visibility && 'visibility',
+      closeTime && 'closeTime',
+      question && 'question',
+      resolvedDescription && 'description',
+    ])
```

Comment:
Currently the contract edits (what shows up in edit history) for descriptions and questions titles are handled via the `on-update-contract.ts` firebase trigger. In the frontend, we're writing to firebase directly to update those fields. The code here would record the title and description edits twice I think.

To solve, you can either remove the insert to `contract_edits` in the firebase trigger and change `saveDescription()` in the frontend to call the api. Or you can not update the question and description here

Reply:
OK, I have removed the recordContractEdit() calls for the question and description.